test/1_unit/005A.stringify.js

```diff
@@ -6,7 +6,7 @@ const tests = [
   { value: null, valueText: 'null', expected: null },
   { value: undefined, valueText: 'undefined', expected: null },
 
```

Comment:
add at least one test with primitive Number and String

Reply:
they are on the line 100